nilearn/_utils/extmath.py

```diff
@@ -75,3 +79,61 @@ def is_spd(M, decimal=15, verbose=1):
     if not ispd and verbose > 0:
         print("matrix has a negative eigenvalue: %.3f" % eigvalsh.min())
     return ispd
+
+
+def check_threshold(threshold, data, percentile_calculate, name='threshold'):
```

Comment:
@GaelVaroquaux currently it is in ```displays.py```. I have to use it outside of plotting modules which means I cannot escape being caught by travis. I am using it to check for valid thresholds in ```threshold_img``` in ```image.py```